src/eureka/S6_planet_spectra/s6_spectra.py

```diff
@@ -201,7 +201,8 @@ def plot_spectra(eventlabel, ecf_path=None, s5_meta=None, input_meta=None):
                 else:
                     # Just load the parameter
                     if meta.sharedp:
-                        meta = parse_s5_saves(meta, log, fit_methods, 'shared')
+                        meta.spectrum_median, meta.spectrum_err = \
+                            parse_s5_saves(meta, log, fit_methods, 'shared')
```

Comment:
There is another instance of this issue at https://github.com/kevin218/Eureka/blob/533cb23a18b0c28a1556fb5ed6c1bf39fa607d22/src/eureka/S6_planet_spectra/s6_spectra.py#L684

Reply:
This will need to be reverted now that my PR has been merged